src/sw.ts

```diff
@@ -201,11 +216,7 @@ self.addEventListener('fetch', (event) => {
  ******************************************************
  */
 async function requestRouting (event: FetchEvent, url: URL): Promise<boolean> {
-  if (!isServiceWorkerRegistrationTTLValid()) {
-    log.trace('Service worker registration TTL expired, unregistering service worker')
-    event.waitUntil(self.registration.unregister())
-    return false
-  } else if (isUnregisterRequest(event.request.url)) {
```

Comment:
moved to after the request to prevent weird network failure issues because sw unregisters in the middle of a response